modules/mediawiki/manifests/init.pp

```diff
@@ -208,8 +217,15 @@
         check_command => 'check_mediawiki!meta.miraheze.org',
     }
 
-    icinga::service { 'php5-fpm':
-        description   => 'php5-fpm',
-        check_command => 'check_nrpe_1arg!check_php_fpm',
-    }
+    if os_version('debian >= stretch') {
+        icinga::service { 'php7.0-fpm':
+            description   => 'php7.0-fpm',
+            check_command => 'check_nrpe_1arg!check_php_fpm',
```

Comment:
Please be aware this nrpe command checks for 'php5-fpm' processes.

Reply:
Yep :).

I am going to rename some things so we can tell apart php5-fpm and php7.0-fpm :).